FiveCalls/FiveCalls/ScheduleReminders.swift

```diff
@@ -158,8 +158,6 @@ struct DayAndTimePickers: View {
             DatePicker("", selection: $selectedTime, displayedComponents: .hourAndMinute)
                 .labelsHidden()
                 .datePickerStyle(WheelDatePickerStyle())
-                .colorInvert() // TODO: this won't work in dark mode
```

Comment:
I can't seem to find a way to change the color of the date picker wheel text that's compatible with dark mode unfortunately so we might have to use defaults here.

Reply:
👍 it looks OK as-is